Formula/convox.rb

```diff
@@ -16,7 +16,7 @@ class Convox < Formula
   def install
     ENV["GOPATH"] = buildpath
     (buildpath/"src/github.com/convox/rack").install Dir["*"]
-    system "go", "build", "-o", "#{bin}/convox", "-v", "github.com/convox/rack/cmd/convox"
+    system "go", "build", "-ldflags=-X main.Version=20160623185456", "-o", "#{bin}/convox", "-v", "github.com/convox/rack/cmd/convox"
```

Comment:
No, version can be either implicit or explicit. Mike removed it because it was explicit without needing to be. It only needs to be written when our self-detection method fails or we're using a custom version for some reason.